src/frontends/pytorch/src/op/pad.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/coordinate_diff.hpp"
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_pad(NodeContext& context) {
+    auto data = context.get_input(0);
+    auto paddings = context.const_input<std::vector<int64_t>>(1);
```

Comment:
`paddings` can be non constant. Did you check if all models we have constant `paddings`?

Reply:
tensor is not acceptable in pytorch according to specification, only tuple of ints expected, so that is why I assume that it is always constant

Reply:
Otherwise, it should be prim::ListConstruct which is not supported on our side yet, probably we can handle this scenario later, if it requires

Reply:
True, but could you check if we have such cases? And how many cases we have?

Reply:
I found only one model family where it is used (7 models) - vit_*_r model from timm, where * is tiny, small, base, large

Reply:
yes, in vit models used ListConstract as input of pad operation, for other cases it is constant